storio-common-annotations-processor/build.gradle

```diff
@@ -1,8 +1,22 @@
 apply plugin: 'java'
+apply plugin: "jacoco"
 
 targetCompatibility = '1.6'
 sourceCompatibility = '1.6'
 
+jacoco {
+    toolVersion = "0.7.4.201502262128"
```

Comment:
Can you please try `0.7.5.201505241946`? See https://github.com/jacoco/jacoco/releases

Also, single quotes :)